homeassistant/components/html5/__init__.py

```diff
@@ -1 +1,50 @@
 """The html5 component."""
+
+import logging
+
+from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
+from homeassistant.const import CONF_PLATFORM, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import config_validation as cv, discovery
+from homeassistant.helpers.typing import ConfigType
+
+from .const import DATA_HASS_CONFIG, DOMAIN
+from .issues import async_create_html5_issue
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORM = Platform.NOTIFY
+CONFIG_SCHEMA = cv.config_entry_only_config_schema(DOMAIN)
+
+
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up the HTML5 push notification component."""
+    hass.data[DATA_HASS_CONFIG] = config
+    existing_config_entry = hass.config_entries.async_entries(DOMAIN)
+    # Iterate all entries for notify to only get HTML5
+    if Platform.NOTIFY in config:
+        for entry in config[Platform.NOTIFY]:
+            if entry[CONF_PLATFORM] == DOMAIN:
```

Comment:
Please reverse the condition and continue when it does not match to reduce code indentation.

Reply:
Adjusted to use .get instead.